README.md

```diff
@@ -1 +1,54 @@
-﻿# Todo List App
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> One paragraph statement about the project.
+
+![screenshot](Screenshottt.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- html
+- css
+- JavaScript
+- Webpack
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
```

Comment:
- [ ] You have a good README but please add commands for installing npm `npm install`, how to start your project in the browser `npm start` and to run the tests as well if you have them `npm test`